src/main/java/seedu/contax/model/ModelManager.java

```diff
@@ -114,6 +116,16 @@ public boolean hasPerson(Person person) {
     @Override
     public void deletePerson(Person target) {
         addressBook.removePerson(target);
+
+        // Delete Successful, dissociate target with any appointments.
+        // The list of matching appointments is cloned because the list iterator is destroyed upon
```

Comment:
```suggestion
        // The list of matching appointments is cloned because the list iterator is destroyed upon delete
```
think you forgot a word here